parser_test.go

```diff
@@ -4605,6 +4615,52 @@ func TestParallel(t *testing.T) {
 	wg.Wait()
 }
 
+func TestCreateTableMultiplePrimaryKey(t *testing.T) {
```

Comment:
Added more test cases and a check in the SQLite to make sure the stmts testes are also not allowed in SQLite.